### PR TITLE
Contact me form

### PR DESCRIPTION
**Pull request by mirouhml**
- Add contact me form.
- Add footer.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -192,6 +192,55 @@ <h2>Skills</h2>
                 </li>
             </ul>
         </section>
+        <section class="contact-me">
+            <form action="https://formspree.io/f/xdobkqjy" method="post">
+                <ul  class="contact-me-form">
+                    <li>
+                        <h1 class="contact-me-form-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
```
Comment (Hamzaoutdoors) [inline index.html]: - [x] the `h1` is the top header of the page and should be used once. `h2-6` are subheaders and can be used repeatedly. So can you please try to change this tag to `h2`, that will make your code more semantic and for good SEO reasons

File: styles.css [hunk 2]
```diff
@@ -396,3 +405,57 @@ ul.information-list-subitems {
...
+  margin-bottom: 73px;
+}
+
+input {
+  border: 0;
+  margin-left: 40px;
+  margin-right: 40px;
+  width: 295px;
+  display: block;
+}
```
Comment (Hamzaoutdoors) [inline styles.css]: - [x]  You need to stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. Please, make sure to give inputs some paddings so they can look closely similar to the [original design ](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471)😸 .

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey team, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README

## Required Changes ♻️

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**N.A**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Whoistolu — APPROVED** [review]
### Hi @mirouhml ,

Your project is complete!  There is nothing else to say other than... It's time to merge.
# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**